### PR TITLE
Capacity FAQ Chinese translation

### PR DESCRIPTION
**Pull request by jl2012**

### CODE REVIEW

**Comment by HeaTeR** [comment]
Is the segregated witness soft fork equivalent to a 4MB block size increase, a 2MB increase, a 1.75MB increase, or what? I keep hearing different numbers.

隔离见证软分叉相当于 4MB、2MB、1.75MB 或究竟多少的区块大小增加？我听到的说法各不相同。

**Comment by jl2012** [comment]
Minor revisions based on @shangzhou 's suggestion.

File: zh_CN/bitcoin-core/capacity-increases-faq.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+
```
Comment (btcdrak) [inline zh_CN/bitcoin-core/capacity-increases-faq.md]: missing `---` to start the YAML header.

Comment (jl2012) [inline zh_CN/bitcoin-core/capacity-increases-faq.md]: Fixed

**Comment by harding** [comment]
Reviewed translations using Google Translate; LGTM.  I'll be merging momentarily.  Thanks!

**Pull request merged**